pom.xml

```diff
@@ -50,6 +51,7 @@
         <converter.version>0.2.2</converter.version>
         <jline.version>2.14.3</jline.version>
         <joda-converter.version>1.7</joda-converter.version>
+        <curator-framework.version>2.9.0</curator-framework.version>
```

Comment:
We finally decide use curator 2.9.0 because is copatible with 3.4.x and 3.5.x zk vesions, and change in the future when zookeeper 3.5.x will be a stable version.